eng/verify-nupkgs.ps1

```diff
@@ -112,4 +116,42 @@ function Unzip {
     [System.IO.Compression.ZipFile]::ExtractToDirectory($zipfile, $outpath)
 }
 
+function Match-VersionAgainstBranch {
+    param ([string]$vsTestVersion, [string]$branchName)
```

Comment:
```suggestion
    param ([string]$vsTestVersion, [string]$branchName, [string[]]$errors)
```

Reply:
Thanks, fixed in follow-up PR, see below.